dspace-xmlui-mirage2/pom.xml

```diff
@@ -53,6 +53,7 @@
                         <exclude>**/readme.txt</exclude>
                         <exclude>**/.bowerrc</exclude>
```

Comment:
Do we still need `.bowerrc`?  Maybe remove here and in `dspace-xmlui-mirage2/src/main/webapp`?

Reply:
Good catch! I've removed them.